pkg/oc/bootstrap/docker/up.go

```diff
@@ -1014,6 +1040,13 @@ func useAnsible(v semver.Version) bool {
 	return v.GTE(openshiftVersion36)
 }
 
+// clusterVersionIsCurrent returns whether the cluster version being
+// brought up matches the client binary being used.  This needs to
+// be updated each release.
+func clusterVersionIsCurrent(v semver.Version) bool {
+	return v.GT(openshiftVersion37)
```

Comment:
Won't this return true for 3.7.1?

Reply:
yeah, good catch, can you address this in your PR? :)